src/test/kotlin/uk/gov/justice/digital/hmpps/manageusersapi/resource/UserControllerIntTest.kt

```diff
@@ -54,13 +57,25 @@ class UserControllerIntTest : IntegrationTestBase() {
           assertThat(it["userMessage"]).isEqualTo("Account for username $username not found")
           assertThat(it["developerMessage"]).isEqualTo("Account for username $username not found")
         }
-      hmppsAuthMockServer.verify(getRequestedFor(urlEqualTo("/auth/api/azureuser/$username")))
+    }
+
+    @Test
+    fun ` should fail with not_found for email address username`() {
+      val username = "testing@digital.justice.gov.uk"
+      externalUsersApiMockServer.stubGetFail("/users/$username", NOT_FOUND)
+      webTestClient.get().uri("/users/$username")
+        .headers(setAuthorisation(roles = listOf()))
+        .exchange()
+        .expectStatus().isNotFound
+      deliusApiMockServer.verify(0, getRequestedFor(urlEqualTo("/users/$username/details")))
+      hmppsAuthMockServer.verify(0, getRequestedFor(urlEqualTo("/auth/api/azureuser/$username")))
```

Comment:
These are just to verify that because it is an email, both delius and auth(for azure users) are not called